src/runtime/hexagon_remote/dlib.cpp

```diff
@@ -508,6 +502,18 @@ void *mmap_dlsym(void *from, const char *name) {
     return (void *)dlib->get_symbol_addr(sym);
 }
 
+// Check all currently loaded libraries for a symbol
+void *mmap_dlsym_libs(const char *name) {
```

Comment:
Is there a fix here, or is it just refactoring?

Reply:
This was refactored in order to expose this functionality so it could be called from halide_remote.cpp to lookup the halide_set_default_thread_priority symbol to see if it is present in the shared runtime.  This was needed to support the non-dlopenbuf path on the 8996/Pixel 1.

Reply:
I see. I think this logic should go in mmap_dlsym, and that should handle the case when the library argument is NULL, or RTLD_SELF, or RTLD_DEFAULT. The design here was that mmap_dlopen/mmap_dlsym/mmap_dlclose mirror the standard APIs.